### ISSUE TITLE
Audit current viewers for accessibility issues

### ISSUE THREAD

**Issue by bibliophileaxe** [Islandora/documentation#2425]

**Comment by joecorall** [comment]
### IIIF
[The accessibility usability community channel in the IIIF slack](https://iiif.slack.com/archives/C0A76RCDYJ0) are working on an accessibility best practices documentation for IIIF manifests that may be helpful with this: [Accessibility Best Practices: IIIF Manifests](https://docs.google.com/document/d/1DFEIfXFZcSV-NgggQpzpe-xuQlxZwg32_FNqID8YQFQ/edit?tab=t.0)

### PDF.js

PDF.js github issues tagged with accessibility: https://github.com/mozilla/pdf.js/issues?q=is%3Aissue%20state%3Aopen%20label%3Aaccessibility

### Mirador
Mirador’s accessibility board: https://github.com/orgs/ProjectMirador/projects/4/views/4
  - Running the Mirador viewer though Wave, Lighthouse, and axe DevTools only showed this issue: https://github.com/ProjectMirador/mirador/issues/4145


### Openseadragon

TBD
